package.json

```diff
@@ -15,10 +15,13 @@
     "@hookform/resolvers": "^3.3.4",
     "@radix-ui/react-label": "^2.0.2",
     "@radix-ui/react-slot": "^1.0.2",
+    "bcrypt": "^5.1.1",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
+    "dotenv": "^16.4.5",
```

Comment:
Why was this added?

Reply:
sorry for the unnecessary addition of this  dotenv and mongodb, was trying something but not required regarding the issue and have been removed